configuration/driver/icedrv_RunMod.F90

```diff
@@ -177,13 +177,18 @@ subroutine ice_step
       if (tr_fsd .and. wave_spec) call step_dyn_wave(dt)
 
       do k = 1, ndtd
-
-        ! ridging
-        call step_dyn_ridge (dt_dyn, ndtd)
-
-        ! clean up, update tendency diagnostics
-        offset = c0
-        call update_state (dt_dyn, daidtd, dvidtd, dagedtd, offset)
+         
+         ! horizontal advection of ice into the single column
+         ! needed if we have closing (otherwise closing produces
+         ! open water)
```

Comment:
Please make the comment a little more terse, e.g. 

`! horizontal advection of ice or open water into the single column`

without the additional explanation (which you've added to the docs, thank you)

Reply:
done